README.md

```diff
@@ -23,19 +23,34 @@ To test the next version using the ArcGIS CDN, simply reference the SDK and CSS
     <link rel="stylesheet" href="https://js.arcgis.com/next/esri/themes/light/main.css">
     <script src="https://js.arcgis.com/next/"></script>
 
+    <!--Add the following script tag when using map components -->
+    <script type="module" src="https://js.arcgis.com/map-components/next/arcgis-map-components.esm.js"></script>
+
 Only production CDN builds are available for download.
 
 ### npm
 
-To test the next version with npm, use `npm install @arcgis/core@next`. For npm, [jsDelivr](https://www.jsdelivr.com/package/npm/@arcgis/core?tab=files) is used to host the `next` CSS:
+The next version is available for testing using npm with local builds.
+
+#### Map components
+
+Run `npm install @arcgis/map-components@next`. For CSS use [jsDelivr](https://www.jsdelivr.com/package/npm/@arcgis/core?tab=files):
+
+    @import "https://cdn.jsdelivr.net/npm/@arcgis/core@next/assets/esri/themes/light/main.css";
+
```

Comment:
Should we also mention the map-component assets?

`https://cdn.jsdelivr.net/npm/@arcgis/map-components@next/dist/arcgis-map-components/assets`

Reply:
Updated, thanks.